integration-test.py

```diff
@@ -359,6 +378,8 @@ def print_coords(f, log, idx, num_tests):
         runpy.run_path(f, init_globals={
             'assert_has_feature': print_coord,
             'assert_no_matching_feature': print_coord,
+            'assert_at_least_n_features': assert_at_least_n_features,
+            'assert_less_than_n_features': assert_less_than_n_features,
```

Comment:
These should be `print_coord`, or similar function with matching signature. This part of the code is used to print the coordinates of all tiles being tested, which can be useful for pre-seeding the dev TOI.